python/lsst/ts/xml/data/sal_interfaces/MTAOS/MTAOS_Events.xml

```diff
@@ -59,6 +59,13 @@
       <Units>micron</Units>
       <Count>50</Count>
     </item>
+    <item>
+      <EFDB_Name>userDoF</EFDB_Name>
+      <Description>User applied offsets.</Description>
+      <IDL_Type>double</IDL_Type>
+      <Units>micron</Units>
```

Comment:
The same question of unit as above. I will ask the user to check the policy file in ts_ofc to know the definition and unit for each DOF.